examples/demo-vendoring/stacks/deploy/prod.yaml

```
@@ -0,0 +1,12 @@
+vars:
+  stage: prod
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)